### PR TITLE
Add DNS over Warp in help

### PR DESCRIPTION
**Pull request by jrf0110**

### CODE REVIEW

File: source/pages/partials/help.pug [hunk 1]
```diff
@@ -23,6 +23,9 @@ section(data-section="help")
       tr
         td Using DNS over TLS (DoT)
         td(data-ref="isDot") Checking...
+      tr
+        td Using DNS over WARP
```
Comment (jrf0110) [inline source/pages/partials/help.pug]: @dknecht Do you mean here?

Comment (dknecht) [reply source/pages/partials/help.pug]: I meant where you add on | off

Comment (jrf0110) [reply source/pages/partials/help.pug]: so the states are `on | off | plus`?

**Comment by dknecht** [comment]
Yep

File: source/pages/partials/help.pug (hunk 1 above)
Comment (dknecht) [reply source/pages/partials/help.pug]: yes

File: source/pages/help.ts [hunk 2]
```diff
@@ -32,6 +32,7 @@ interface TraceInfo {
   spdy: string
   http: string
   loc: string
+  warp?: 'off' | 'on'
```
Comment (dknecht) [inline source/pages/help.ts]: Here ^

**Pull request merged**